drivers/net/ethernet/intel/idpf/eth/idpf_ethtool.c

```diff
@@ -256,7 +281,7 @@ static int idpf_set_channels(struct net_device *netdev,
 	num_req_tx_q = ch->combined_count + ch->tx_count;
 	num_req_rx_q = ch->combined_count + ch->rx_count;
 
-	dev = &vport->adapter->pdev->dev;
+	dev = &eth_shared->pdev->dev;
```

Comment:
use a macro like idpf_pdev_dev(vport) to go from vport to the device pointer of the PCI device .
This comment applies to all usages of pdev->dev

Reply:
I think this has been addressed as part of pdev changes.